platform/solr/platform-solr-server-standalone/src/main/resources/solr/conf/schema.xml

```diff
@@ -687,450 +590,461 @@
-        <!--</analyzer>-->
+      <!--<analyzer>-->
+        <!--<tokenizer class="solr.StandardTokenizerFactory"/>-->
+        <!--<filter class="solr.LowerCaseFilterFactory"/>-->
+        <!--<filter class="solr.StopFilterFactory" ignoreCase="true" words="lang/stopwords_es.txt" format="snowball" />-->
+        <!--<filter class="solr.SpanishLightStemFilterFactory"/>-->
+        <!--&lt;!&ndash; more aggressive: <filter class="solr.SnowballPorterFilterFactory" language="Spanish"/> &ndash;&gt;-->
+      <!--</analyzer>-->
     <!--</fieldType>-->
 
     <!--&lt;!&ndash; Basque &ndash;&gt;-->
+    <!--<dynamicField name="*_txt_eu" type="text_eu"  indexed="true"  stored="true"/>-->
     <!--<fieldType name="text_eu" class="solr.TextField" positionIncrementGap="100">-->
-        <!--<analyzer>-->
-            <!--<tokenizer class="solr.StandardTokenizerFactory"/>-->
-            <!--<filter class="solr.LowerCaseFilterFactory"/>-->
-            <!--<filter class="solr.StopFilterFactory" ignoreCase="true" words="lang/stopwords_eu.txt" />-->
-            <!--<filter class="solr.SnowballPorterFilterFactory" language="Basque"/>-->
-        <!--</analyzer>-->
+      <!--<analyzer>-->
+        <!--<tokenizer class="solr.StandardTokenizerFactory"/>-->
+        <!--<filter class="solr.LowerCaseFilterFactory"/>-->
+        <!--<filter class="solr.StopFilterFactory" ignoreCase="true" words="lang/stopwords_eu.txt" />-->
+        <!--<filter class="solr.SnowballPorterFilterFactory" language="Basque"/>-->
+      <!--</analyzer>-->
     <!--</fieldType>-->
 
     <!--&lt;!&ndash; Persian &ndash;&gt;-->
+    <!--<dynamicField name="*_txt_fa" type="text_fa"  indexed="true"  stored="true"/>-->
     <!--<fieldType name="text_fa" class="solr.TextField" positionIncrementGap="100">-->
-        <!--<analyzer>-->
-            <!--&lt;!&ndash; for ZWNJ &ndash;&gt;-->
-            <!--<charFilter class="solr.PersianCharFilterFactory"/>-->
-            <!--<tokenizer class="solr.StandardTokenizerFactory"/>-->
-            <!--<filter class="solr.LowerCaseFilterFactory"/>-->
-            <!--<filter class="solr.ArabicNormalizationFilterFactory"/>-->
-            <!--<filter class="solr.PersianNormalizationFilterFactory"/>-->
-            <!--<filter class="solr.StopFilterFactory" ignoreCase="true" words="lang/stopwords_fa.txt" />-->
-        <!--</analyzer>-->
+      <!--<analyzer>-->
+        <!--&lt;!&ndash; for ZWNJ &ndash;&gt;-->
+        <!--<charFilter class="solr.PersianCharFilterFactory"/>-->
+        <!--<tokenizer class="solr.StandardTokenizerFactory"/>-->
+        <!--<filter class="solr.LowerCaseFilterFactory"/>-->
+        <!--<filter class="solr.ArabicNormalizationFilterFactory"/>-->
+        <!--<filter class="solr.PersianNormalizationFilterFactory"/>-->
+        <!--<filter class="solr.StopFilterFactory" ignoreCase="true" words="lang/stopwords_fa.txt" />-->
+      <!--</analyzer>-->
     <!--</fieldType>-->
 
     <!--&lt;!&ndash; Finnish &ndash;&gt;-->
```

Comment:
I recommend being brutal in removing commented out stuff you don't need (particularly what came in Solr's \_default that you don't need).  You can always refer back to the Solr ref guide and/or provide a link to the \_default schema on GitHub.